ivy/functional/backends/paddle/manipulation.py

```diff
@@ -378,11 +374,21 @@ def tile(
             shape = paddle_backend.multiply(x.shape, repeats).tolist()
         return paddle.zeros(shape).cast(x.dtype)
 
-    if x.dtype in [paddle.int8, paddle.int16, paddle.uint8, paddle.float16]:
-        return paddle.tile(x.cast("float32"), repeats).cast(x.dtype)
     return paddle.tile(x, repeats)
 
 
+@with_unsupported_dtypes(
+    {
+        "2.5.1 and below": (
+            "bfloat16",
+            "float16",
+            "int8",
+            "int16",
+            "uint16",
```

Comment:
ig this should be `uint8` instead of `uint16`

Reply:
yep, It's a typo error. A sharp reviewer as usual 🚀

Reply:
thanks for fixing!